griptape/tokenizers/openai_tokenizer.py

```diff
@@ -120,9 +120,9 @@ def count_tokens(self, text: str | list[dict], model: Optional[str] = None) -> i
                 return self.count_tokens(text, model="gpt-4-0613")
             else:
                 raise NotImplementedError(
-                    f"""token_count() is not implemented for model {model}. 
-                    See https://github.com/openai/openai-python/blob/main/chatml.md for 
-                    information on how messages are converted to tokens.""",
+                    f"""token_count() is not implemented for model {model}.
+                    See https://github.com/openai/openai-python/blob/main/chatml.md for
+                    information on how messages are converted to tokens."""
```

Comment:
Not exactly related to this, but won't `f"""` include all of the indentation whitespace? Would it be better to rely on the automatic concatenation of consecutive string literals?

For example:
```py
raise NotImplementedError(
    f"token_count() is not implemented for model {model}. "
    "See https://github.com/openai/openai-python/blob/main/chatml.md for "
    "information on how messages are converted to tokens."
)
```